### DISCUSSION TITLE
Screen Reader Accessibility Seems to be Broken

### DISCUSSION THREAD

**Discussion by kvutter** [home-assistant/frontend#18901] [Accessibility]
Hi all! I use a screenreader(VoiceOver for Mac and iOS, and NVDA for Windows) to use HA. Its been working great since I started in July 2023. 

But on 10.5 and Front-end 20231030.2, many menu items, e.g., the settings menu items are all identified as “Group” rather than their titles.  I can only access the menu items by memory of their order.  Sometimes I can use an OCR to read the entire screen, get the order of items, then choose the right one.  This happens on Mac, Windows, and even in the iOS app.  There are also some dialogues, e.g., the create automation where you choose a blank one or something else, which no longer can be read.  These used to work fine.  

In the past, there were only a few things I couldnt access, e.g., the HA Terminal for SSH.   Now, there are quite a few more things that are nearly impossible on Mac and iOS, and quite difficult on Windows also.  If there’s more info from me that would be helpful, please let me know.  I would certainly love to be able to get around HA as easily as I used to.  Thank you for your consideration.

Kevin

**Comment by steverep** [comment]
@kvutter please see my comments on #18759 regarding the Safari issues.  This is because of a Webkit bug that showed up in Safari/iOS 17.  There's not much I can do about it.  Nothing in HA is causing it.  My workaround for now is to temporarily turn on "screen recognition" in VoiceOver.

You mentioned some things being more difficult in Windows too.  Please do file issues for anything specific and I will take a look.  The discussion board is mainly used for feature requests.
